src/components/utils/Markdown.tsx

```diff
@@ -66,84 +67,101 @@ const CopyCodeButton = ({ code }: { code: ReactNode[] }) => {
       <CopyIcon boxSize={4} mr={1} /> {copied ? "Copied!" : "Copy Code"}
     </Button>
   );
-};
+});
+
+const Codeblock = memo(function Codeblock({ className, inline, children, ...props }: {
+  className?: string;
+  inline?: boolean;
+  children: ReactNode[];
+}) {
```

Comment:
idt memoizing here helps because it wont do a deep compare on children? correct me if im wrong

Reply:
It's true.
> [From PR] Use memo for the remaining components in the file; not entirely necessary, but the performance impact of memo is negligible and prevents annoying to debug re-render issues.

Happy to strip them if you've got a preference to remove them.

Reply:
> [E.g. the coinbase way](https://attardi.org/why-we-memo-all-the-things/)

oo this post and perspective is interesting — though [kentcdodds seems to disagree](https://twitter.com/kentcdodds/status/1452716184869109760?s=20) (however my guess is he just read the title as this post doesnt rly address these points head on lol)

i think we should keep the diff as minimal as possible for this PR and just not memo these functions — but would be open to discussing in an issue whether we should adopt the coinbase way going forward

Reply:
Appreciate the follow up post - tend to agree it doesn't address the points. I think it's a React idiosyncrasy that makes the performance overhead minimal (kentcdodds says so himself a few times "incidentally [the performance hit] is probably happening anyway") and is definitely not good advice generally.

Anyway for the purpose of this PR: I'll strip them out tonight 😄

Reply:
Done!